yt_dlp/extractor/vimeo.py

```diff
@@ -354,15 +362,22 @@ def _fetch_oauth_token(self, client):
 
         return f'Bearer {self._oauth_tokens[cache_key]}'
 
-    def _call_videos_api(self, video_id, unlisted_hash=None, path=None, *, force_client=None, query=None, **kwargs):
-        client = force_client or self._configuration_arg('client', [self._DEFAULT_CLIENT], ie_key=VimeoIE)[0]
+    def _get_requested_client(self):
```

Comment:
@Maxr1998 just an fyi, now with this change you could write an extractor plugin that overrides `_get_requested_client` to force a particular client (or just directly overwrite this method in your library's code)